package.json

```diff
@@ -82,7 +78,7 @@
     "onchange": "^3.3.0",
     "postcss-loader": "^2.0.9",
     "postcss-object-fit-images": "^1.1.2",
-    "prettier": "^1.9.2",
+    "prettier": "1.7",
```

Comment:
Could you please explain a bit more why the `prettier` downgrade is necessary?

Reply:
Leftovers of some fighting, yesterday :) Changed it.

Reply:
Narrowed it down a bit, but after 1.10 prettier tries to put empty spaces when we have label of the sort `label: card--shadow-${shadow};`, right before the $.